dotcom-rendering/src/components/Caption.tsx

```diff
@@ -102,8 +102,8 @@ const veryLimitedWidth = css`
 `;
 
 const captionPadding = css`
-	padding-left: 8px;
-	padding-right: 8px;
+	padding-left: 10px;
+	padding-right: 10px;
```

Comment:
No Source spacing there?

Reply:
Afraid not, [it needs 10px to align](https://github.com/guardian/dotcom-rendering/blob/d710b666246dd08b7c471f890e984ba52b8185b5/dotcom-rendering/src/components/Section.tsx#L170) with the rest of the body content which is within a section component